src/Application/Console/Formatters/FormatResolver.php

```diff
@@ -29,22 +28,28 @@ public static function resolve(
         OutputInterface $output,
         OutputInterface $consoleOutput
     ): Formatter {
-        $requestedFormat = $input->getOption('format');
+        $requestedFormats = $input->getOption('format');
 
-        if (! is_string($requestedFormat)) {
-            throw new InvalidArgumentException(
-                'Format has to be a string.'
-            );
+        if (! is_array($requestedFormats)) {
+            $consoleOutput->writeln('<fg=red>Could not understand requested format, using fallback [console] instead.</>');
+            $requestedFormats = ['console'];
         }
 
-        $requestedFormat = strtolower($requestedFormat);
+        $formatters = [];
+        foreach ($requestedFormats as $requestedFormat) {
+            if (class_exists($requestedFormat)) {
```

Comment:
we should check also if $requestedFormat implement `NunoMaduro\PhpInsights\Application\Console\Contracts\Formatter` ?

Reply:
I don't think that is needed tbh. It will just add another condition in our code which doesn't really benefit that much. 
However I can add it in if you feel it is needed 👍

Reply:
I think it's better to be defensive here, as we permit to pass Custom formatter, we should be sure when the class will be used, it respect our contract.

Reply:
Alright, what do you say to the new implementation?